jenkins/spark-nightly-build.sh

```diff
@@ -95,6 +95,12 @@ for buildver in "${SPARK_SHIM_VERSIONS[@]:1}"; do
     $MVN -U -B clean install -pl '!tools' $MVN_URM_MIRROR -Dmaven.repo.local=$M2DIR \
         -Dcuda.version=$CUDA_CLASSIFIER \
         -Dbuildver="${buildver}"
+    # enable UTF-8 and run regular expression tests
+    env LC_ALL="en_US.UTF-8" $MVN verify -pl '!tools' $MVN_URM_MIRROR -Dmaven.repo.local=$M2DIR \
```

Comment:
I tried this with `mvn test` but it would never run the integration test that are also necessary to run. This way it runs the 3 scalatests and the integration test as well.

Reply:
Actually we run integration test in https://github.com/NVIDIA/spark-rapids/blob/branch-22.08/jenkins/spark-tests.sh as downstream testing of our nightly build to cover all test scenarios (spark releases+OS, over 10 pipelines)